bin/aapp_dr_runner.py

```diff
@@ -360,8 +349,8 @@ def read_arguments():
     else:
         args.station = args.station.lower()
     if not args.environment:
-        print ("Environment required! " +
-               "Use command-line switch -e <environment> e.g. de, test")
+        print("Environment required! " +
+              "Use command-line switch -e <environment> e.g. de, test")
         sys.exit()
```

Comment:
This can be more compactly written as

`sys.exit("Environment required!" + ...)`

this will write the text to stderr and exit with an error code, which would probably be better.

Reply:
Ah, thanks, didn't know that. Again code I had not been touching.